(blog): Integrate empress-blog and Netlify CMS (#668)
ember-learn/ember-blog: source/2020-06-05-the-ember-times-issue-151.md

---

## [Blog post: How to integrate Netlify-CMS with Empress-blog 🚀](https://www.mylittletechlife.com/integrate-netlify-cms-with-empress-blog)

> [Robin Philip Thomas](https://github.com/MalayaliRobz) wrote a blog on integrating empress-blog and Nelify CMS.

Reviewer: ```suggestion
[Robin Philip Thomas](https://github.com/MalayaliRobz) wrote a blog on integrating empress-blog and Netlify CMS.
```

> [empress-blog](https://github.com/empress/empress-blog) is designed to be a fully-functional, static site implementation of a blog system built on Ember with fully working out of the box SEO friendly output based on markdown. [Netlify CMS](https://www.netlifycms.org/) is based on client-side javascript, reusable APIs and prebuilt markup.

Reviewer: I'm not too familiar with Empress so I wasn't sure how to parse the first sentence. I wondered if making a list might help clarify what its features are.

If I understand you right, maybe:
```suggestion
What is [empress-blog](https://github.com/empress/empress-blog)?

- A fully-functional, static site implementation of a blog system
- Built on Ember
- Fully working out of the box
- SEO friendly output based on Markdown

[Netlify CMS](https://www.netlifycms.org/) is based on client-side JavaScript, reusable APIs and prebuilt markup.
```

Author: Ah! That sounds perfect @ijlee2 :)
